packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -326,17 +326,20 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
           this._closeHttpSpan(span);
         });
         response.on('error', (error: Err) => {
```

Comment:
Please note that adding an `error` listener modifies the application behavior. see [nodejs docs](https://nodejs.org/dist/latest-v16.x/docs/api/events.html#events_error_events) for details.

I don't think that monitoring tools like OTel should have a such a side effect.